### PR TITLE
Fix return types and annotations on FakeAIOKafkaConsumer's topics

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Fixes https://github.com/alm0ra/mockafka-py/issues/139

### CODE REVIEW

File: mockafka/aiokafka/aiokafka_consumer.py [hunk 1]
```diff
@@ -111,8 +111,8 @@ async def commit(self):
 
         self.consumer_store = {}
 
-    async def topics(self) -> list[str]:
-        return self.subscribed_topic
+    async def topics(self) -> set[str]:
+        return set(self.subscribed_topic)
```
Comment (PeterJCLaw) [inline mockafka/aiokafka/aiokafka_consumer.py:115]: @alm0ra something I wondered about here is whether this should return all the topics from the store, not just the current subscriptions. I think that could be seen as closer to the underlying behaviour, which fetches all the topics (that the client has permission to see) at this point (https://github.com/aio-libs/aiokafka/blob/e2afc034b59626242b51b06b30a62c565ef491d9/aiokafka/consumer/consumer.py#L631-L638).

Comment (alm0ra) [reply mockafka/aiokafka/aiokafka_consumer.py:115]: Hey Peter,

Thanks for your contribution! I was reviewing the code, and I noticed a comment at this line:

`Get all topics the user is authorized to view.`

I'm a bit confused about what this means. Does it refer to the consumer subscribing to specific topics, or is it retrieving all existing topics in Kafka that the user has access to?

Comment (alm0ra) [reply mockafka/aiokafka/aiokafka_consumer.py:115]: I wasn't familiar with Kafka ACLs before, but after looking into it, it seems like Kafka does have access control mechanisms to manage user permissions for accessing topics. You can read more about it here: [Kafka ACL Overview](https://docs.confluent.io/platform/current/security/authorization/acls/overview.html).

Given that we don't use ACLs in our setup, I believe the code here should simply retrieve all topics that exist in our Kafka store.

**Review by alm0ra — CHANGES_REQUESTED** [review]

File: mockafka/aiokafka/aiokafka_consumer.py (hunk 1 above)
Comment (PeterJCLaw) [reply mockafka/aiokafka/aiokafka_consumer.py:115]: Yeah, I think that would be a reasonable approach here.

**Comment by PeterJCLaw** [comment]
@alm0ra what changes are you requesting here?

File: mockafka/aiokafka/aiokafka_consumer.py (hunk 1 above)
Comment (alm0ra) [reply mockafka/aiokafka/aiokafka_consumer.py:115]: What do you think about merging this pr and opening another issue about return all topics in FakeAioKafkaConsumer?

**Review by alm0ra — APPROVED** [review]

File: mockafka/aiokafka/aiokafka_consumer.py (hunk 1 above)
Comment (PeterJCLaw) [reply mockafka/aiokafka/aiokafka_consumer.py:115]: -> #141

**Pull request merged**